popsycle/synthetic.py

```diff
@@ -2045,20 +2125,21 @@ def reduce_blend_rad(blend_tab, new_blend_rad, output_root, overwrite=False):
 ############################################################################
 
 
-def convert_photometric_99_to_nan(table):
+def convert_photometric_99_to_nan(table, photometric_system='ubv'):
     for name in table.colnames:
-        if ('ubv' in name) or ('exbv' in name):
+        if ('exbv' in name) or (photometric_system in name):
             cond = table[name] == -99
```

Comment:
just curious... what does cond = table[name] == -99 do? 
is this better than np.where?

Reply:
Functionally it's no different than np.where. np.where returns a list of indices only at the location of the condition, while this way return a list of booleans at every location in the array.

If you would rather we could switch this to `cond = np.where(table[name] == -99)[0]` and it would have an identical result.

Reply:
no this is fine, i'd just not seen it before

Reply:
Yea, I have no idea when to use one versus the other!  ‾\_(ツ)_/‾

Reply:
Not critical: I'd be interested if there is any performance difference.

Reply:
It looks like np.where is **slightly** slower, but barely:
```
In [153]: arr = np.arange(10000000)
In [154]: %timeit cond = arr % 2 == 0
231 ms ± 3.53 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
In [155]: %timeit np.where(arr % 2 == 0)
257 ms ± 5.39 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

Reply:
Lets go with
cond = np.where(table[name] == -99)[0]

Every little bit helps!

Reply:
You got it!